core/src/it/scala/org/ensime/fixture/AnalyzerFixture.scala

```diff
@@ -17,7 +17,17 @@ object AnalyzerFixture {
   private[fixture] def create(search: SearchService)(implicit system: ActorSystem, config: EnsimeConfig, vfs: EnsimeVFS): TestActorRef[Analyzer] = {
     val indexer = TestProbe()
     val projectActor = TestProbe()
-    TestActorRef(Analyzer(projectActor.ref, indexer.ref, search))
+    TestActorRef(Analyzer(projectActor.ref, indexer.ref, search, EnsimeProject(
```

Comment:
this doesn't look right. Was this a workaround to the transitive classpath problem?